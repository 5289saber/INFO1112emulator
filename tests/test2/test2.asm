SET 50 1
SET 51 255

SET 52 254
SET 53 2

SET 54 128
SET 55 128

SET 56 255
SET 57 0

ADD 58 50 51
ADD 59 52 53
ADD 60 54 55
ADD 61 56 57

SET 62 'l'

PUT 58 N d
PUT 58 N h
PUT 62 S 1

PUT 59 N d
PUT 59 N h
PUT 62 S 1

PUT 60 N d
PUT 60 N h
PUT 62 S 1

PUT 61 N d
PUT 61 N h
//PUT 62 S 1