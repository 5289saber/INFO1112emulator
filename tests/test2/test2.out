Finished compiling. File tests/test2/test2.bin has been created.
Binary files tests/test2/test2.bin and tests/test2/test2Exp.bin differ
